index.html

```diff
@@ -1,12 +1,476 @@
+                <div class="content">
+                    <div class="flex">
+
+                        <div class="column">
+                            <div class="column-img">
+                                <img src="img/illustration-contact-us.svg" />
+                            </div>          
+                        </div>
+
+                        <div class = "column flex-column">
+                            <div class="column-title">
+                                <h2>Contact Us</h2>
+                            </div>
+                            <div class="column-desc">
+                                <p>We believe in the power of conversations. Start one with us today!</p>
+                            </div>
+
+                            <div class="form">
+
+                                <div class="form-flex">
+
+                                    <div class="form-column flex-column ">
+                                        <div class="column-field">
+                                            <input class="field" type="text" placeholder="Input name"> </input>
+                                        </div>
+                                        <div class="column-field">
+                                            <input class ="field" type="text" placeholder="Zip code"> </input>
+                                        </div>
+                                    </div>
+                                    <div class="form-column flex-column ">
+                                        <div class="column-field">
+                                            <input class="field" placeholder="Email"> </input>
+                                        </div>
+                                        <div class="column-field">
+                                            <input class="field" placeholder="Company"> </input>
+                                        </div>
+                                    </div>
+                                    
+                                </div>
+
+                                <div class="form-column-mesg">
+                                    <textarea class="column-textarea" placeholder="Message"></textarea>
+                                </div>
+                            </div>
+                            <div class="column-btn">
+                                <button class="blue-btn"><a class="blue-btn-title">Get in Touch</a></button>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+
+                
+            </div>
+        </section>
+
+    </main>
+
+    <footer class="footer">
+        <div class="footer-top">
+            <div class="container">
+                <div class="content">
+                    <div class="content-img">
+                        <img src= "./img/logo-white.svg"/>
+                    </div>
+
+                    <div class="content-two-columns ">
```

Comment:
avoid such frequent usage of `two-columns` keyword. There are a lot of different ways to describe layout properties. It can be a problem when some keyword is extremely frequented through the project.